indidriver.c

```diff
@@ -2313,13 +2366,12 @@ void IDSetLight(const ILightVectorProperty *lvp, const char *fmt, ...)
     printf("  timestamp='%s'\n", timestamp());
     if (fmt)
     {
-        va_list ap;
-        va_start(ap, fmt);
         char message[MAXINDIMESSAGE];
-        printf("  message='");
         vsnprintf(message, MAXINDIMESSAGE, fmt, ap);
+
+        // #PS: why not escapeXML_fputs?
+        printf("  message='");
         printf("%s'\n", entityXML(message));
-        va_end(ap);
     }
```

Comment:
Sometimes the "entityXML" function is used,
sometimes "escapeXML_fputs", but nothing here.
Right?

With the next review, I can improve it.